appcelerator/titanium_mobile
# fix(android): openPhotoGallery() crashes when canceled as of 9.1.0

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-28178

**Test:**
1. Build and run `PhotoVideoSelectTest.js` attached to [TIMOB-28178](https://jira.appcelerator.org/browse/TIMOB-28178) on Android.
2. When dialog is shown, tap on "Photo".
3. Back out of the app.
4. Verify app does not crash and you return to an empty window.
5. Exit and relaunch the app.
6. Select "Photo" from the dialog.
7. Select an image from the gallery.
8. Verify Titanium app displays that photo.
9. Exit and relaunch the app.
10. Select "Video" from the dialog.
11. Select a video from the gallery.
12. Verify Titanium app plays the selected video.
